rollup.config.ts

```diff
@@ -0,0 +1,126 @@
+import { babel } from '@rollup/plugin-babel'
+import { nodeResolve } from '@rollup/plugin-node-resolve'
+import fs from 'fs-extra'
+import glob from 'glob'
+import path from 'path'
+import shorthash from 'shorthash'
+import type { AddAttributesToSVGElementPlugin, RemoveAttrsPlugin, RemoveViewBoxPlugin } from 'svgo'
+
+import { svg } from './plugins/svg'
+
+const svgConfig = {
+  plugins: [
+    {
+      name: 'removeAttrs',
+      params: {
+        attrs: 'fill',
+      },
+    } as RemoveAttrsPlugin,
+    { active: false, name: 'removeViewBox' } as RemoveViewBoxPlugin,
+    {
+      name: 'addAttributesToSVGElement',
+      params: {
+        attributes: [
+          {
+            fill: 'currentColor',
+          },
+        ],
+      },
+    } as AddAttributesToSVGElementPlugin,
+  ],
+}
+
+export default () => {
+  const modulesPathname = path.join(__dirname, 'modules')
+
+  const modules = glob
+    .sync(path.join(__dirname, 'svg/*.svg'))
+    .reduce<Array<{ code: string; filename: string; moduleFilename: string; modulePathname: string }>>(
```

Comment:
Possible d'extraire le type pour que ce soit un peu plus lisible ?
```suggestion
    .reduce<Array<ModuleType>>(
```

Reply:
Good point ! Je fais le changement